### PR TITLE
Patrick/python/lab06

### PR DESCRIPTION
**Pull request by PatrickCoteWurzler**
lab06 ready for grading

### CODE REVIEW

File: code/patrick/python/lab06.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    
+    return numbers
+
+
+
+
+
+matches = 0
+balance = 0
+def num_matches(winning_ticket, ticket):
```
Comment (chapman311) [inline code/patrick/python/lab06.py:35]: This function has repeating code that can be cleaned up with a for loop.

File: code/patrick/python/lab06.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    
+     matches = 0
+     balance = 0
```
Comment (chapman311) [inline code/patrick/python/lab06.py:38]: Unused variable `balance`. Sets `balance` to 0. Not sure why this is here.

File: code/patrick/python/lab06.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+for _ in range(0, 100000):
+     user_numbers = pick6()
+     winning_numbers = pick6()
+     
+     matches = num_matches(user_numbers, winning_numbers)
+    
+   
+     if user_numbers != winning_numbers:
+        balance += -2
+     if matches == 1:
```
Comment (chapman311) [inline code/patrick/python/lab06.py:69]: Start of repeating code that could all be nested under a single line statement using `matches`.

**Review by chapman311 — CHANGES_REQUESTED** [review]
Lab 6 version 1 works and meets lab requirements. 

Version 2 is missing. Please refer to the lab's instructions and add the required Version 2.

**Review by chapman311 — APPROVED** [review]
Version 1 and Version 2 are both looking good. Thank you for adding the version 2 file.

**Review by pjz987 — APPROVED** [review]

**Pull request merged**